irohad/consensus/yac/transport/impl/network_impl.hpp

```diff
@@ -101,6 +102,8 @@ namespace iroha {
          * Subscriber of network messages
          */
         std::weak_ptr<YacNetworkNotifications> handler_;
+        std::shared_ptr<network::AsyncGrpcClient<google::protobuf::Empty>>
```

Comment:
Please, add empty line and a short comment, describing purpose of `async_call` variable to be consistent with the rest of the file